tests/mobile/workorder-workflow.spec.js

```diff
@@ -13,95 +12,55 @@ const workflowTitle = prefix + 'workflow';
-      return core.workflows.create(workflowTitle)
-        .then(wf => {
-          workflow = wf;
-          return core.workflows.addStep(vehicleStepTitle, 'vehicleInspection', wf.id)
-        })
-        .then(() => core.users.filter(authData.users.TREVER.username))
-        .then(u => core.workorders.create(workorderTitle, u.users[0].id, workflow.id))
-        .then(wo => workorder = wo);
-    });
-
-    step('login to mobile as ' + authData.users.TREVER.username, function() {
-      authService.openMobileApp();
-      browser.driver.executeScript(function() {
-        sessionStorage.clear();
-        localStorage.clear();
-      });
-      authService.openMobileApp();
-      authService.verifyLoginPageIsVisible();
-      authService.loginToMobileApp(authData.users.TREVER.username,
-        authData.password.DEFAULT_PASSWORD);
-    });
+describe('MOBILE E2E', function() {
 
-    step('verify workorders page is displayed', function() {
-      authService.verifySuccessfulLogin();
-    });
+  let workorder;
+  let workflow;
 
-    step('select a workorder from list', function() {
-      browser.sleep(20000);
-      workorderService.selectWorkorderFromTheList(workorderTitle);
-    });
-
-    step('that workorder details are present and correct', function() {
-      browser.sleep(1000);
-      const workorderDetails = {
-        id: workorder.id,
-        status: 'New',
-        workorderName: workorderTitle
-      };
-      workorderService.verifyWorkorderDetailsArePresentAndCorrect(workorderDetails);
-    });
-
-    step('that the begin workflow button is present', function() {
-      workorderService.verifyWorkorderWorkflowIsNotCompleted();
-    });
-
-    step('click on the begin workflow button', function() {
-      workorderService.beginWorkflow();
-    });
-
-    step('verify elements of workflow completeion page are visible', function() {
-      workflowService.verifyWorkflowFormIsVisible();
-    });
-
-    step('click back button', function() {
-      workflowService.cancelWorkflowChanges();
-    });
-
-    step('verify that the begin workflow button is still present', function() {
-      browser.sleep(1000);
-      workorderService.verifyWorkorderWorkflowIsNotCompleted();
-    });
-
-    step('click on begin workflow button', function() {
-      workorderService.beginWorkflow();
-    });
-
-    step('click continue button', function() {
-      browser.sleep(10000);
-      workflowService.submitWorkflowDetails();
-    });
-
-    step('verify workflow section details are correct', function() {
-      browser.sleep(1000);
-      workorderService.verifyWorkorderWorkflowIsCompleted();
-    });
+  beforeAll(function() {
+    return core.auth.login(authData.users.DAISY.username, authData.password.DEFAULT_PASSWORD)
+      .then(() => core.cleanup(prefix))
+      .then(() => core.workflows.create(workflowTitle))
+      .then(wf => {
+        workflow = wf;
+        return core.workflows.addStep(vehicleStepTitle, 'vehicleInspection', wf.id);
```

Comment:
Two comments here:

1. It might possible to export these functions, albeit difficult, in order to match the syntax to the rest of the test file.   

2. On line 25 - `workflow = wf` on line 26 `wf.id` -- workflow does not deem to be used until line 29. It would be better to use it here as workflow is assigned from the result (wf). It would then be more consistent